flink-connectors/flink-connector-aws-kinesis-data-streams/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ConsumerConfigConstants.java

```diff
@@ -18,23 +18,21 @@
 package org.apache.flink.streaming.connectors.kinesis.config;
 
 import org.apache.flink.annotation.PublicEvolving;
-import org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer;
-import org.apache.flink.streaming.connectors.kinesis.internals.ShardConsumer;
 import org.apache.flink.streaming.connectors.kinesis.model.SentinelSequenceNumber;
 
-import com.amazonaws.services.kinesis.model.ShardIteratorType;
+import software.amazon.awssdk.http.Protocol;
 
 import java.time.Duration;
 
 /**
- * Optional consumer specific configuration keys and default values for {@link
+ * Optional consumer specific configuration keys and default values for {@code
  * FlinkKinesisConsumer}.
  */
 @PublicEvolving
 public class ConsumerConfigConstants extends AWSConfigConstants {
```

Comment:
This class is used by `AwsV2Util` which is used by the new features

Reply:
SentinelSequenceNumber
SequenceNumber are used by this so also needed

Reply:
> This class is used by AwsV2Util which is used by the new features

It should not be, since this class is `ConsumerConfigConstants` and your new feature is Producer. Both sequence number classes are also consumer concerns